CHANGES.md

```diff
@@ -2,6 +2,8 @@
 
 ## Version 0.8.8
 - scriptpubkey_csv_2of3_then_2_from_bytes has been removed from the API.
+- psbt: psbt_extract now takes an extra flags parameter. The Java and Python
```

Comment:
this seems out of place? commit is about `psbt_finalize`?

Reply:
Yes good catch, c'n'p from a previous API change, will update.